### PR TITLE
HW-46700 - Show minimum required java version

### PR DESCRIPTION
**Pull request by wmews-hw**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ Hyperwallet REST SDK (Beta)
 
 A library to manage users, transfer methods and payments through the Hyperwallet API
 
+Prerequisites
+------------
+
+Hyperwallet's Java server SDK requires at minimum JDK (Java Development Kit) version 1.7 and above.
```
Comment (yherasym-hw) [inline README.md]: do we build 2 SDK's: one with Java7 and 2nd one with Java8?

Comment (wmews-hw) [reply README.md]: No we just build this project with jdk 1.7 as per what we can see on pom.xml:

```
<plugin>
        <groupId>org.apache.maven.plugins</groupId>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>2.3.2</version>
        <configuration>
              <source>1.7</source>
              <target>1.7</target>
              <encoding>UTF-8</encoding>
         </configuration>
</plugin>
```

Comment (yherasym-hw) [reply README.md]: @wmews-hw i think that added statement is not properly tight to what use said; 
"Hyperwallet's Java server SDK is building with java 1.7 that is allowing clients to use Java 1.7+" - something like this;

Comment (wmews-hw) [reply README.md]: The section talks about Prerequisites and in my understand it has to explicit say which minimum java version is required. Nothing more than that.

Comment (yherasym-hw) [reply README.md]: the statement allows to switch to Java 1.8 without any notification that could be a risk;

Comment (wmews-hw) [reply README.md]: If any other developer clones this project and updates it to 1.8 it is totally accepted. Once they want to do their own implementation they become responsible for that, including testing.
Btw... I don't see any risk on bumping it to 1.8. The problem could be by decreasing it to 1.6, 1.5...

**Pull request merged**